src/include/ipc/header.h

```diff
@@ -104,6 +104,8 @@
 #define SOF_IPC_PM_CORE_ENABLE			SOF_CMD_TYPE(0x007)
```

Comment:
Use "include:" prefix in commit subject.

Reply:
ack